setup/local/README.md

```diff
@@ -0,0 +1,419 @@
+# A Connaisseur Setup Guide
+
+## Intro
+
+Hey! Pssst, over here. I've got something for you. Image signature verification for Kubernetes, that only allows cryptographically signed images to be deployed to the cluster as a means to prevent malicious containers from entering the system, such as those pesky bitcoin miners. Sounds complicated? It ain't, so let's go!
+
+It all starts with Kubernetes, the thing that is taking the tech world by storm. Everyone is talking about it, everyone is using it and it handles important questions such as *"can we automate it?"* or *"is it scalable?"*.
+
+What about security? Is Kubernetes doing anything regarding security? Well yes, it does. There is Role-Based Access Control (RBAC) available, which can be fine-tuned to your need. You can set up network restrictions, restrict API access and so on. Additionally, there exist tools around Kubernetes such as *Istio* that can be used for handling traffic encryption of cluster-internal and outgoing communications.
+
+What about image integrity, making sure that running applications don't do some bitcoin mining in addition to or instead of showing your *"Hello World!"* page? No? Well then let's change that!
+
+Entering the scene – **Connaisseur**. It runs as a Mutating Admission Webhook that intercepts incoming requests and changes all images to their signed versions, if present. Otherwise Connaisseur denies the request. To do so, Connaisseur requires a **Notary server**, which is an implementation of *The Update Framework*. Notary stores the actual signatures and relevant metadata of the tag-to-digest relation.
+
+All you need to do is set those two up and you get image integrity for your Kubernetes cluster. This write-up shall give you a practical setup guide that shows how to install Notary and Connaisseur in your cluster and how to sign all your images.
```

Comment:
```suggestion
# Local Setup Guide
This guide describes setting up a completely local installation of Connaisseur using minikube paired with a local Harbor installation that combines registry and notary servers. This setup is a bit more cumbersome since it does not use existing public infrastructure such as PKI. However, it is entirely based on open-source solutions, does not require continuous access to the internet and you control every part of the installation. This makes it specifically suited e.g. for security researchers and users interested in the inner workings of DCT and Connaisseur.

>  Note: For quick testing, we recommend using the simple main [setup guide](setup/README.md) using Docker Hub.
```